src/containers/pages/InterventionPlan/IRS/tests/IRS.test.tsx

```diff
@@ -64,7 +65,6 @@ describe('containers/pages/IRS', () => {
     );
     const helmet = Helmet.peek();
     expect(helmet.title).toEqual(IRS_PLANS);
-    expect(toJson(wrapper)).toMatchSnapshot();
```

Comment:
Agreed with @moshthepitt, let's add this line back but find some child of wrapper which is most relevant to the tests (so we're not saving a snapshot of the whole app).